potpie/cli.py

```
@@ -0,0 +1,313 @@
+        return False
+    except PermissionError:
+        return True
+    return True
+
+
+def _matches_tracked_process(tracked_process: TrackedProcess) -> bool:
+    if not _is_running(tracked_process.pid):
+        return False
+
+    try:
+        command = subprocess.check_output(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)